tools/serverpod_cli/lib/src/analyzer/entities/checker/checker.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:serverpod_cli/src/analyzer/entities/validation/keywords.dart';
+import 'package:yaml/yaml.dart';
+
+class Checker {
```

Comment:
This name feels a bit too generic for a class as it could be a checker of anything.

Reply:
fair